saturn-rstudio/Dockerfile

```diff
@@ -0,0 +1,51 @@
+ARG SATURN_R_IMAGE
+FROM ${SATURN_R_IMAGE}
+
+ARG RSTUDIO_VERSION=2021.09.0-351
+
+ARG TINI_VERSION=0.18.0
+
+# hadolint ignore=DL3008,DL3009
+RUN sudo apt-get update --fix-missing \
+    && sudo update-locale LANG=en_US.utf8 \
+    && sudo apt-get install -y --no-install-recommends \
```

Comment:
I would suggest to add cleanup for apt stuff after installs.

Reply:
Good call.